try { fn:error() } catch * { 0 }, try { fn:error() } catch err:FOER0000 { 1 }
